.changeset/wet-dolphins-love.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-github': major
+---
+
+Adding description in function 'onTeamEditedInOrganization` in event of team.edited
```

Comment:
This is a bit too internal for anyone not looking at the code. Think of changesets as the text that end users read in the changelog for this package.

May I suggest this instead but feel free to tweak it :)
 
```suggestion
Updated the `team.edited` event emitted from `GithubOrgEntityProvider` to also include teams description.
```